### DISCUSSION TITLE
Request: Full text search in attachment files

### DISCUSSION THREAD

**Discussion by kaori-tokashiki** [growilabs/growi#7723] [Ideas / Requests]
## Informations
- [20211008_添付ファイル内の全文検索](https://dev.growi.org/Web会議室/20211008_添付ファイル内の全文検索)

**Comment by yuki-takei** [comment]
This request is related to the feature idea [**Full-Text Search + RAG Answers #11104**](https://github.com/growilabs/growi/discussions/11104) proposed by the GROWI development team, which covers broader search improvements including full-text search capabilities.

Please upvote and share your thoughts there!
